scripts/mkimg.arm.sh

```diff
@@ -86,6 +94,7 @@ profile_rpi() {
 	case "$ARCH" in
 		armhf) kernel_flavors="rpi rpi2";;
 		armv7) kernel_flavors="rpi2";;
+		aarch64) kernel_flavors="rpi rpi4";;
```

Comment:
@filoozom shouldn't armv7 also include RPi4 ? ARMv7 is the only "really" supported arch for the RPi4. 
`armv7) kernel_flavors="rpi2";;` -> `armv7) kernel_flavors="rpi2 rpi4";;`